wp-content/themes/core/components/share/Share_Controller.php

```diff
@@ -114,7 +114,7 @@ private function get_data(): array {
 			$query = get_search_query();
 
 			$data['link']  = $this->normalize_url( get_search_link( $query ) );
-			$data['title'] = sprintf( __( 'Search Results: %s', 'tribe' ), esc_attr( $query ) );
+			$data['title'] = sprintf( esc_html__( 'Search Results: %s', 'tribe' ), esc_attr( $query ) );
```

Comment:
it is always good to include a comment directed at translators when using `sprint()`.  That way the translator doesn't have to know code to figure out what they are translating. 

Here is an [example from the WP handbook](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#variables).

Reply:
Yes, good point. I updated with a comment.